lib/src/main/java/co/nimblehq/alpine/lib/mrz/MrzManager.kt

```diff
@@ -0,0 +1,145 @@
+@file:JvmName("MrzManager")
+
+package co.nimblehq.alpine.lib.mrz
+
+import android.graphics.BitmapFactory
+import android.util.Log
+import co.nimblehq.alpine.lib.mrz.MrzProcessorException.*
+import com.google.mlkit.common.MlKitException
+import com.google.mlkit.vision.common.InputImage
+import com.google.mlkit.vision.text.*
+import com.google.mlkit.vision.text.latin.TextRecognizerOptions
+import java.io.File
+import java.util.regex.Pattern
+
+interface ResultListener {
```

Comment:
In addition to @Wadeewee's comment, this `ResultListener` can be changed to `MrzManagerResult`. What do you think?

Reply:
@lydiasama changed to `MrzProcessorResult` instead.